migrations/sql/V2019.02.11.10.23__permit_amendments.sql

```diff
@@ -0,0 +1,161 @@
+alter table permit drop constraint permit_pkey;
+ALTER TABLE permit ADD COLUMN permit_id SERIAL PRIMARY KEY;
+alter table mine_party_appt drop constraint mine_party_appt_permit_party_fk;
+
+
+CREATE TABLE permit_amendment_type_code
+(
+    permit_amendment_type_code character varying(2) NOT NULL,
+    description character varying(100) NOT NULL,
+    display_order smallint,
+    active_ind boolean DEFAULT true NOT null,
+    create_user character varying(60) NOT NULL,
+    create_timestamp timestamp
+    with time zone DEFAULT now
+    () NOT NULL,
+    update_user character varying
+    (60) NOT NULL,
+    update_timestamp timestamp
+    with time zone DEFAULT now
+    () NOT NULL
+);
+
+ALTER TABLE permit_amendment_type_code OWNER TO mds;
+ALTER TABLE ONLY permit_amendment_type_code
+
+ADD CONSTRAINT permit_amendment_type_code_pkey PRIMARY KEY
+(permit_amendment_type_code);
+
+INSERT INTO permit_amendment_type_code
+	(
+	permit_amendment_type_code,
+	description,
+	display_order,
+	create_user,
+	update_user
+	)
+VALUES
+	('AM', 'Permit Amendment', 10, 'system-mds', 'system-mds'),
+	('ML', 'Amalgamated Permit', 20, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+CREATE TABLE permit_amendment_status_code
+(
+	permit_amendment_status_code character varying(2) NOT NULL,
+	description character varying(100) NOT NULL,
+	display_order smallint,
+	active_ind boolean DEFAULT true NOT null,
+	create_user character varying(60) NOT NULL,
+	create_timestamp timestamp with time zone DEFAULT now() NOT NULL,
+	update_user character varying(60) NOT NULL,
+	update_timestamp timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE permit_amendment_status_code OWNER TO mds;
+ALTER TABLE ONLY permit_amendment_status_code
+ADD CONSTRAINT permit_amendment_status_code_pkey PRIMARY KEY
+(permit_amendment_status_code);
+
+INSERT INTO permit_amendment_status_code
+	(
+	permit_amendment_status_code,
+	description,
+	display_order,
+	create_user,
+	update_user
+	)
+VALUES
+	('A', 'Active', 10, 'system-mds', 'system-mds'),
```

Comment:
This is a minor comment re: status values. My recommendation would be to use 3-character codes rather than 1. The reason is we want the codes to be able to properly reflect the value. If we use "A" for active, we can't use "A" for anything else (i.e. "Accepted")... this would mean needed to use another letter which may not make sense. If we used 3-character codes, we could have "ACT" for Active and "ACC" for Accepted (just as an example)... and these codes could bring more value to the business.

Reply:
Sure, I'm always happy to be more verbose 👍

Reply:
Updated with three letter codes.

Reply:
Looks great! Thanks :)